macSKK/StateMachine.swift

```diff
@@ -773,6 +800,19 @@ class StateMachine {
         }
     }
 
+    private func useKanaRuleIfPresent(inputMode: InputMode, romaji: String, input: String) -> Romaji.ConvertedMoji? {
+        if inputMode != .direct && !romaji.isEmpty {
+            let converted = kanaRule.convert(romaji + input)
+            if converted.kakutei != nil && converted.input == "" {
+                return .some(converted)
+            } else {
+                return .none
+            }
+        } else {
+            return .none
```

Comment:
いつか.some/.noneにするかもしれませんが、いまのところnilを使っているので統一してほしいです。

```suggestion
            if converted.kakutei != nil && converted.input == "" {
                return converted
            } else {
                return nil
            }
        } else {
            return nil
```

Reply:
なるほど、Swiftって`nil`と`.none`/`.some`にに互換性があるのか 🙆‍♀️